environment-template/software/main.tf

```diff
@@ -8,9 +8,12 @@ module "software" {
   sql_user                            = data.terraform_remote_state.infrastructure.outputs.sql_user
   sql_db_pw                           = data.terraform_remote_state.infrastructure.outputs.sql_db_pw
   sql_ip                              = data.terraform_remote_state.infrastructure.outputs.sql_ip
-  sql_database                        = data.terraform_remote_state.infrastructure.outputs.sql_database
+  sql_hono_database                   = data.terraform_remote_state.infrastructure.outputs.sql_hono_database
+  sql_grafana_database                = data.terraform_remote_state.infrastructure.outputs.sql_grafana_database
   service_name_communication          = data.terraform_remote_state.infrastructure.outputs.service_name_communication
   device_communication_static_ip_name = data.terraform_remote_state.infrastructure.outputs.device_communication_static_ip_name
+  grafana_static_ip_name              = try(data.terraform_remote_state.infrastructure.outputs.grafana_static_ip_name[0], "")
```

Comment:
Can you please cxplain why the grafana_static_ip_name property is an array?

Reply:
The grafana static IP resource is created conditionally by setting count to 0 or 1 and this leads to the output being an array.

Reply:
Okay, that make sense!